src/app/view/components/dot-container-selector/dot-container-selector.component.spec.ts

```diff
@@ -62,12 +65,31 @@ describe('ContainerSelectorComponent', () => {
                     name: 'Container 2',
                     type: 'Container',
                     source: CONTAINER_SOURCE.FILE,
-                    path: 'container/path'
+                    path: 'container/path',
+                    parentPermissionable: {
+                        hostname: 'demo.dotcms.com'
+                    }
                 }
             ];
         })
     );
 
+    it('should show the hots name and container name',
+        fakeAsync(() => {
```

Comment:
fakeAsync not needed

Reply:
done https://github.com/dotCMS/core-web/pull/1419/commits/9f0329227f49d5e2ba679ec450bf7170112f69e3#diff-f80340d948cb93edaa053f4b5ac39829L78